cmd/describe_config.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	e "github.com/cloudposse/atmos/internal/exec"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// describeComponentCmd describes configuration for components
+var describeConfigCmd = &cobra.Command{
+	Use:                "config",
+	Short:              "describe config",
+	Long:               `This command shows CLI configuration`,
+	FParseErrWhitelist: struct{ UnknownFlags bool }{UnknownFlags: true},
+	Run: func(cmd *cobra.Command, args []string) {
+		err := e.ExecuteDescribeConfig(cmd, args)
+		if err != nil {
+			color.Red("%s\n", err)
+			fmt.Println()
+			os.Exit(1)
```

Comment:
I think eventually we need to make this more DRY — seems to be happening every time we handle an error

Reply:
yes
but it looks different on the screen, so it's not every time we add an empty line